package.json

```diff
@@ -96,7 +96,6 @@
     "progress": "^2.0.3",
     "prop-types": "^15.6.2",
     "realm-network-transport": "^0.7.0",
-    "request": "^2.88.0",
```

Comment:
Did you run `npm install`? I would expect the package-lock.json to update as well after this change.

Reply:
package-lock.json added.  Thanks :)